.ci/jobs/e2e-testing-helm-daily-mbp.yml

```
@@ -0,0 +1,44 @@
+---
+- job:
+    name: e2e-tests/e2e-testing-helm-daily-mbp
+    display-name: End-2-End tests for Observability Helm charts Pipeline
+    description: Run E2E Helm Charts test suite daily, including maintenance branches
+    view: E2E
+    project-type: multibranch
+    pipeline-scm:
+      script-path: .ci/e2eTestingHelmDaily.groovy
+      scm:
+        - github:
+            branch-discovery: no-pr
+            head-filter-regex: '(master|\d+\.\d+\.x)'
+            discover-pr-forks-strategy: merge-current
+            discover-pr-forks-trust: permission
+            discover-pr-origin: merge-current
+            discover-tags: false
+            notification-context: 'beats-ci/e2e-testing'
```

I guess, the notification for this MBP should not happen since it's an orchestrator.

See `disable-pr-notifications` in https://docs.openstack.org/infra/jenkins-job-builder/project_workflow_multibranch.html#project_multibranch.github_scm
```suggestion
            notification-context: 'beats-ci/e2e-testing'
            disable-pr-notifications: true
```